ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_node_type.schema.yml

```diff
@@ -225,6 +225,16 @@ $defs:
               Set point-to-Point interface speed for the uplink switch interface only.
               Speed should be set in the format `<interface_speed>` or `forced <interface_speed>` or `auto <interface_speed>`.
             type: str
+          uplink_mtu:
+            documentation_options:
+              table: node-type-uplink-configuration
+            description: Point-to-Point uplinks MTU in bytes. This configuration over-rides `p2p_uplinks_mtu` configuration.
```

Comment:
```suggestion
            description: Point-to-Point uplinks MTU in bytes. This setting overrides the `p2p_uplinks_mtu` setting.
```

Reply:
Fixed.

Reply:
you missed a `the`

Reply:
Sorry, I overlooked the suggestion. Corrected now.